### PR TITLE
Update README.md with Patrick Lumumba's details

### PR DESCRIPTION
**Pull request by lumumbapl**
Hello,

This pull request seeks to add my details into the list of WordPress contributors available for sponsorship opportunities. 

Here are the details. 

[Patrick Lumumba](https://wpcorner.co/author/patrick-l/)
  - Twitter: [@lumiblog](https://twitter.com/lumiblog)
  - WordPress: [@pedromendonca](https://profiles.wordpress.org/lumiblog/)
  - Sponsor: [GitHub Sponsor Program](https://github.com/sponsors/lumumbapl)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,11 @@ Kevin Batdorf
   - WordPress: [@meaganhanes](https://profiles.wordpress.org/meaganhanes)
   - Sponsor: [GitHub Sponsor Program](https://github.com/sponsors/meaganhanes)
 
+[Patrick Lumumba](https://wpcorner.co/author/patrick-l/)
+  - Twitter: [@lumiblog](https://twitter.com/lumiblog)
+  - WordPress: [@pedromendonca](https://profiles.wordpress.org/lumiblog/)
```
Comment (jeffpaul) [inline README.md:119]: looks like the handle should be `@lumiblog` not `@pedromendonca`

Comment (lumumbapl) [reply README.md:119]: Yes. It is @lumiblog. 

Thanks, @jeffpaul

**Comment by aurooba** [comment]
Thank you for reviewing this @jeffpaul!

**Comment by jeffpaul** [comment]
@aurooba I didn't have write access to this repo or the PR branch, so couldn't push a fix but the displayname for the WP account still needs to be updated

**Pull request merged**